docs/guides/rtc_product_guide.md

```diff
@@ -1,30 +1,93 @@
 # Sentinel-1 RTC Product Guide
+This document is a guide for users of Radiometrically Terrain Corrected (RTC) Sentinel-1 products processed by the Alaska Satellite Facility (ASF).
 
-This document is a guide for users of radiometrically terrain-corrected Sentinel-1 products processed by the Alaska Satellite Facility.
+## Introduction
+
+### Sentinel-1 Mission
+The [Sentinel-1 mission](https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1) collects C-band band SAR from a pair of polar-orbiting satellites launched by the European Space Agency (ESA) as part of the [Copernicus program](http://www.esa.int/Applications/Observing_the_Earth/Copernicus/Overview4). The Sentinel-1A satellite was launched April 3, 2014, and the Sentinel-1B satellite was launched April 25, 2016.
+
+The two Sentinel-1 satellites each have a 12-day repeat cycle, but their orbits are offset 180 degrees so that one or the other will pass over the same location on earth every 6 days. Most areas of the earth will still only have imagery collected every 12 days, but Europe and select areas of interest are imaged with a 6-day interval, as described in the [mission observation scenario](https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1/observation-scenario).
+
+Because this is a polar-orbiting satellite constellation, areas near the poles may have a number of overlapping paths, resulting in even more frequent acquisitions with similar footprints. 
+
+The relatively short interval between acquisitions makes this SAR dataset a very useful tool for monitoring rapid or sudden landscape changes. In addition, SAR can image the earth's surface through cloud or smoke cover and does not require sunlight, so valid imagery can be collected on every pass. This is particularly useful for monitoring conditions during natural disasters such as hurricanes or wildfires, or in areas that are prone to frequent cloud cover.
+
+### SAR Distortions
```

Comment:
This header might be better as `Correcting for SAR Distortions`, or something similar?  The whole page is about RTC, but there's no header or table of contents entry that calls out "here's what RTC is"

Reply:
@asjohnston-asf I adjusted the headers in the Terrain Correction workflow to reflect the radiometric part of the process. Does that address your concerns? Another approach would be to add a bit of a high-level description of RTC in that very first blurb under the Product Guide title.
@talogan can you take a look to make sure those headers are appropriate? I also added a reference to the DEM mapping function, as we didn't really say anywhere in the workflow description how the DEM was used in the process. Amend as suitable.

Reply:
The headers look good to me.